jerry-core/debugger/jerry-debugger.h

```diff
@@ -21,6 +21,12 @@
 
 #define MAX_BUFFER_SIZE 128
 
+static uint8_t jerry_debugger_buffer[MAX_BUFFER_SIZE];
```

Comment:
To avoid duplicate definitions this should only be an extern definition:

    extern uint8_t jerry_debugger_buffer[MAX_BUFFER_SIZE];

In the C file you should create the variable:

    uint8_t jerry_debugger_buffer[MAX_BUFFER_SIZE];

Also comment about the variable is mission.